backend/app/main.py

```
@@ -242,6 +242,23 @@ async def lifespan(app: FastAPI):
+    # This enables DingTalk, Feishu, WeChat bot integrations
+    logger.info("Initializing IM Channel Manager...")
+    from app.services.channels import get_channel_manager
+
+    channel_manager = get_channel_manager()
+    db = SessionLocal()
+    try:
+        started_count = await channel_manager.start_all_enabled(db)
+        logger.info(
+            f"✓ IM Channel Manager initialized, {started_count} channels started"
+        )
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)